src/APP/pages/aboutUs/sections/PartnerCTA.jsx

```diff
@@ -56,6 +56,7 @@ function PartnerCTA() {
           <p>
             <a
               href="/src/assets/reports/spaceyatech-internship-program.pdf"
+              download="SpaceYaTech_Internship_Program_2023.pdf"
```

Comment:
I removed this on purpose. it's preventing the file from opening on a new tab first. Lawrence and Weshy complained about this behavior this week.
```suggestion

```